### PR TITLE
Added Swagger documentation API

### PR DESCRIPTION
**Pull request by pbteja1998**
* [x] Added `Swagger Documentation API`
* [x] All APIs that are implemented can be viewed in `/api-docs` route
* [x] Changed Contributing Guidelines to add about Swagger
* [x] Removed `API.md` file

### CODE REVIEW

File: webapp/coderplex_apis/urls.py [hunk 1]
```diff
@@ -19,11 +19,13 @@
...
 from rest_framework_jwt.views import refresh_jwt_token
 from rest_framework_jwt.views import verify_jwt_token
+from rest_framework_swagger.views import get_swagger_view
+schema_view = get_swagger_view(title='CoderPlex APIs')
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^', include('books.urls')),
-    url(r'^$', views.HelloWordView.as_view()),
+    url(r'^$', schema_view),
```
Comment (rrmerugu) [inline webapp/coderplex_apis/urls.py]: add the docs route to something like "api-docs", the way you implemented, makes the api documentation as default route of api.coderplex

**Review by rrmerugu — CHANGES_REQUESTED** [review]

File: webapp/coderplex_apis/urls.py (hunk 1 above)
Comment (pbteja1998) [reply webapp/coderplex_apis/urls.py]: Ok, will change this

**Comment by pbteja1998** [comment]
What about API.md now ? Should I remove it ?

File: webapp/coderplex_apis/urls.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^', include('books.urls')),
-    url(r'^$', schema_view),
+    url(r'^$', views.HelloWordView.as_view()),
+    url(r'^api-docs$', schema_view),
```
Comment (rrmerugu) [inline webapp/coderplex_apis/urls.py]: looks good, are you able to access the api docs at `http://localhost:8000/api-docs` ? if yes, we are good with this.

Comment (pbteja1998) [reply webapp/coderplex_apis/urls.py]: Yes, I am able to access them at `http://localhost:8000/api-docs`

Comment (rrmerugu) [reply webapp/coderplex_apis/urls.py]: perfect then, if you make the changes for `Changed Contributing Guidelines to add about Swagger` , its good to go for a merge.

Comment (pbteja1998) [reply webapp/coderplex_apis/urls.py]: what about `API.md` ? Should I keep it or remove it?

Comment (rrmerugu) [reply webapp/coderplex_apis/urls.py]: I don't see any necessity for 2 Api documentations, its confusing to know which one is up to date. swagger anyways, is live documentation is obviously up to date with out any changes, so you can remove API.md

**Review by rrmerugu — APPROVED** [review]

**Pull request merged**